config/nvim/init.vim

```diff
@@ -1,3 +1,93 @@
     set runtimepath^=~/.vim runtimepath+=~/.vim/after
     let &packpath = &runtimepath
     source ~/.vim/vimrc
+
+if has('nvim-0.5')
```

Comment:
Does this do >0.5, like the normal vim "contains feature number" thing?

Reply:
Yep, it should work for Neovim 0.5 and everything later.